### DISCUSSION TITLE
Package with custom terminal?

### DISCUSSION THREAD

**Discussion by ghost** [pyinstaller/pyinstaller#5793] [Features & Development]
Hi!

Been reading over the docs and didnt see anything regarding this, mightve missed something! 

Is/could it be possible to build a exe with a custom terminal? WSL/CMDer ect?

**Comment by bwoodsend** [comment]
I'm guessing you mean change which terminal is opened when you double-click the app built by PyInstaller. In which case the answer is no. Applications (built by PyInstaller or otherwise) don't have any control over which terminal is used. That's all controlled by Windows. See #5602.
